lib/src/components/NxList/__tests__/NxList.test.tsx

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { ReactElement } from 'react';
+import { NxLoadError, NxLoadingSpinner } from '../../..';
+import NxList from '../NxList';
+import { NxListProps } from '../types';
+import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import { act } from 'react-dom/test-utils';
+import { mount, ReactWrapper, shallow } from 'enzyme';
+
+describe('NxList', function() {
+
+  let mountContainers: HTMLElement[] = [];
+
+  function getMountContainer() {
+    const newContainer = document.createElement('div');
+    mountContainers.push(newContainer);
+    document.body.append(newContainer);
+
+    return newContainer;
+  }
+
+  // create a mounted wrapper that is attached to the document, and deal with the timing complexities
+  // triggered by NxList's MutationObserver usage
+  async function mountAttached(jsx: ReactElement) {
+    let retval: ReactWrapper;
+    await act(async () => {
+      retval = mount(jsx, { attachTo: getMountContainer() });
+    });
+    retval!.update();
+
+    return retval!;
+  }
+
+  afterEach(function() {
+    mountContainers.forEach(container => document.body.removeChild(container));
+    mountContainers = [];
+  });
+
+  const minimalProps: NxListProps = {};
+  const getShallow = getShallowComponent(NxList, minimalProps);
+
+  it('renders a list', function() {
+    const nxList = getShallow();
+    expect(nxList).toMatchSelector('.nx-list');
+  });
+
+  it('renders the classNames given to it', function() {
+    const extendedProps: Partial<NxListProps> = {
+      className: 'test-classname ufo'
+    };
+    const nxList = getShallow(extendedProps);
+    expect(nxList).toMatchSelector('.nx-list.test-classname.ufo');
+  });
+
+  it('passes props to NxList correctly', function() {
```

Comment:
I don't think this is actually testing anything aside from enzyme.  You could replace `NxList` with literally any component here and (typescript notwithstanding) it would pass

Reply:
> I don't think this is actually testing anything aside from enzyme. You could replace `NxList` with literally any component here and (typescript notwithstanding) it would pass

This test has been removed. The test 'renders a bulleted list correctly' tests if prop-passing is done correctly anyway, so should account for this test as well.
55b92e6395a57fc06d88def0b018b36a0a874305